deepmedic/frontEnd/trainSession.py

```diff
@@ -195,7 +195,7 @@ def run_session(self, *args):
             self._log.print3("=======================================================\n")
 
             do_training(*([sessionTf, saver_all, cnn3d, trainer] + self._params.get_args_for_train_routine()),
-                        tensorboard_loggers=tensorboard_loggers)
+                        tensorboard_loggers=tensorboard_loggers, norm_params=self._params.get_norm_params())
```

Comment:
I dont think they should be separate from the rest parameters returned by  .get_args_for_train_routine().

Put them to be returned in there.